locales/ki.json

```diff
@@ -1,16 +1,37 @@
 {
-	"successfully sent query": "Ubutumwa bwoherejwe neza",
-	"successfully sent": "Igikorwa cyagenze neza",
-	"welcome": "Murakaza neza kuri Barefoot Nomad",
-	"successfully logged in": "Winjiye neza",
-	"Email not found": "Imeri ntabwo yabonetse",
+	"Successfully sent query": "Ubutumwa bwoherejwe neza",
+	"Successfully sent": "Igikorwa cyagenze neza",
+	"Welcome": "Murakaza neza kuri Barefoot Nomad",
+	"You signed up successfully": "Wiyandikishije neza",
+	"This Email is in use": "Iyi Emeri iri gukoreshwa",
+	"This Username is in use": "Iri zina  rirakoreshwa",
+	"A verification email has been sent to ": "Imeri yo kugenzura yoherejwe kuri ",
+	"The account has been verified. Please log in.": "Konti yagenzuwe. Nyamuneka injira.",
+	"This user has already been verified.": "Uyu mukoresha yamaze kugenzurwa.",
+	"we are unable to find a user with that email!": "Ntidushobora kubona umukoresha ufite iyo imeri!",
+	"This account has  already been verified.Please log In": "Iyi konti imaze kugenzurwa. Nyamuneka injira.",
+	"No account associated to this email": "Nta konti ijyanye n'iyi imeri",
+	"The link to reset password sent succesfully to your email!": "Ihuza ryo gusubiramo ijambo ryibanga ryoherejwe kuri imeri yawe!",
 	"Wrong Password": "Ijambobanga ritari ryo",
+	"Password don't match": "Ijambo ryibanga ntirihuye",
+	"password updated succesfully!": "Ijambo ryibanga ryavuguruwe neza!",
+	"Confirmation email resend succesfully!": "Imeri yemeza kohereza neza!",
 	"Email is required": "Imeri irakenewe",
 	"Password is required": "Ijambobanga rirakenewe",
-	"Password must be in the range of 6 to 15 characters": " Ijambobanga rigomba kuba riri hagati yinyuguti 6 kugeza kuri 15",
-    "Your account has not been verified": "Konti yawe ntabwo yagenzuwe",
-    "updatedSuccessfully": "Imyirondoro yawe yahinduwe neza.",
-    "updateFailed": "Guhindura imyorondoro ntibikunze, mugerageze mukanya.",
-    "userNameTaken": "Izina [userName] ryafashwe.",
-    "userNotFound": "Nta konti bihuye dufite!" 
-}
+	"Fullname is required": "Amazina arakenewe",
+	"Your account has not been verified": "Konti yawe ntabwo yagenzuwe",
+	"Email not found": "Imeri ntiyabonetse",
+	"User logged in successfully": "Umukoresha yinjiye neza",
+	"Username is required": "Izina ry'ukoresha rirakenewe",
+	"Username must be in the range of 6 to 15 characters": "Izina ryukoresha rigomba kuba riri hagati yinyuguti 6 kugeza kuri 15.",
+	"ConfirmPassword is required": "kwemeza ijambo ryibanga birakenewe",
+	"Password must be in the range of 6 to 15 characters": "Ijambobanga rigomba kuba riri hagati y’inyuguti 6 kugeza kuri 15.",
+	"This is not a valid email address format ": "Ubu ntabwo ari imiterere ya imeri yemewe ",
+	"User Not Found": "Umukoresha Ntabonetse",
+	"Requested user": "Umukoresha usabwa",
+	"Updated Successfully": "Yavuguruwe neza",
+	"Update Failed": "Ivugurura ryanze",
+	"Username has been taken": "Izina ry'ukoresha ryafashwe",
```

Comment:
I think the removed Kinyarwanda translations for `user not found`, `updated successfully`, `update failed`, and `username has been taken` are more understandable to the user than their replacements. So, no need for changing them!